package.json

```diff
@@ -35,72 +38,68 @@
+    "Rick Markins <rmarkins@gmail.com>"
+  ],
+  "files": [
+    "dist",
+    "src",
+    "README.md",
+    "index.d.ts"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/react-dropzone/attr-accept/issues"
   },
   "homepage": "https://github.com/react-dropzone/attr-accept#readme",
   "devDependencies": {
-    "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.10.5",
-    "@babel/plugin-external-helpers": "^7.2.0",
-    "@babel/plugin-proposal-do-expressions": "^7.6.0",
-    "@babel/plugin-proposal-export-default-from": "^7.5.2",
-    "@babel/plugin-proposal-logical-assignment-operators": "^7.2.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@babel/plugin-proposal-pipeline-operator": "^7.5.0",
-    "@babel/plugin-transform-runtime": "^7.6.2",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-react": "^7.6.3",
-    "@babel/register": "^7.6.2",
-    "@commitlint/cli": "^8.2.0",
-    "@size-limit/preset-small-lib": "^2.1.6",
-    "babel-eslint": "^8.0.0",
-    "babel-loader": "^8.0.0",
-    "babel-preset-env": "^1.6.0",
-    "core-js": "^3.1.4",
-    "cross-env": "^6.0.3",
-    "cz-conventional-changelog": "^1.2.0",
-    "eslint": "^4.6.1",
-    "eslint-config-airbnb": "^15.1.0",
-    "eslint-config-okonet": "^6.1.3",
-    "eslint-config-prettier": "^2.4.0",
-    "eslint-plugin-flowtype": "^2.35.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-node": "^5.1.1",
-    "eslint-plugin-prettier": "^2.2.0",
-    "eslint-plugin-react": "^7.3.0",
-    "expect": "^1.20.2",
-    "husky": "^0.14.3",
-    "lint-staged": "^4.1.3",
-    "mocha": "^3.5.0",
-    "prettier": "^1.6.1",
-    "rimraf": "^3.0.0",
-    "webpack": "^4.0.0",
-    "webpack-cli": "^3.3.12"
+    "@babel/core": "^7.25.8",
+    "@commitlint/cli": "^19.5.0",
+    "@eslint/js": "^9.12.0",
+    "@size-limit/preset-small-lib": "^11.1.6",
+    "browserslist": "^4.24.0",
+    "cz-conventional-changelog": "^3.3.0",
+    "eslint": "^9.12.0",
+    "eslint-plugin-import": "^2.31.0",
+    "expect": "^29.7.0",
+    "husky": "^9.1.6",
+    "lint-staged": "^15.2.10",
+    "mocha": "^10.7.3",
+    "parcel": "^2.12.0",
+    "rimraf": "^6.0.1",
+    "typescript": "^5.6.3"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=18"
   },
   "config": {
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
   "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "git add"
+    "*.mjs": [
+      "eslint --fix"
     ]
   },
   "size-limit": [
     {
-      "path": "dist/index.js",
+      "path": "dist/index.mjs",
+      "limit": "2 KB"
+    },
+    {
+      "path": "dist/index.cjs",
       "limit": "2 KB"
     }
-  ]
+  ],
+  "browserslist": [
+    "defaults and fully supports es6-module",
+    "maintained node versions"
+  ],
+  "targets": {
+    "types": false
+  },
+  "@parcel/resolver-default": {
```

Comment:
We could have probably continued using babel/webpack and slimmed down on some of deps (since most of the functionality should be available now).

Mostly becayue this would be a new system that's not used in the other repos and it would be good to stick with the same tooling across. I've made an attempt to use https://swc.rs/ in the other repo, maybe we could use that unless there's a strong case for parcel (I'm unfamiliar with it).